index.html

```diff
@@ -19,31 +17,31 @@ <h1>COLORANDOM</h1>
                     <div class="color-box" id="b1">
                         <img src="./assets/unlocked.png" class="lock-box hover unlocked">
                     </div>
-                    <div class="hex-code">#EA9999</div>
```

Comment:
Part of iteration 0 is having these values hard coded. I know that's never going to show up anymore, but I'm not 100% sure if we should take them out due to the spec calling for it.

Reply:
Never mind, re-read the spec, the spec says 'for now'. So it's meant to be refactored and removed. Nice work.